sop/modules/OSD/partials/deploy_toolbox.adoc

```diff
@@ -1,11 +1,17 @@
 
-*Deploy Toolbox*
+*Deploy and Access the Toolbox*
 
 If the toolbox is not deployed, please follow the directions below.
 
-.Step 1: Deploy Rook-Ceph toolbox if needed:
+.Deploy Rook-Ceph toolbox if needed:
 [source,role="execute"]
 ----
 oc patch OCSInitialization ocsinit -n openshift-storage --type json --patch  '[{ "op": "replace", "path": "/spec/enableCephTools", "value": true }]'
 ----
 
+.Run the following to rsh to the toolbox pod:
+[source,role="execute"]
+----
+TOOLS_POD=$(oc get pods -n openshift-storage -l app=rook-ceph-tools -o name)
```

Comment:
StorageCluster will need to be patched for toolbox to be deployed.
oc patch ocsinitialization ocsinit -n openshift-storage --type json --patch  '[{ "op": "replace", "path": "/spec/enableCephTools", "value": true }]
'

Reply:
Line 9. I will double check it is displaying properly.